### DISCUSSION TITLE
5244端口不行换成8088可以

### DISCUSSION THREAD

**Discussion by miao5748** [AlistGo/alist#3206] [Q&A] [answered]
INFO[2023-01-31 15:41:43]/source/cmd/server.go:40 github.com/alist-org/alist/v3/cmd.glob..func5() start server @ 0.0.0.0:5244
FATA[2023-01-31 15:41:43]/source/cmd/server.go:51 github.com/alist-org/alist/v3/cmd.glob..func5.1() failed to start: listen tcp 0.0.0.0:5244: bind: An attempt was made to access a socket in a way forbidden by its access permissions.

**Comment by miao5748** [comment, marked as answer]
命令行  netsh winsock reset   解决

**Comment by QiuYeDx** [comment]
什么原理，好像真的有用
